manager/integration/negative_tests/negative.robot

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Documentation    Negative Test Cases
+
+Library    ../framework/steps.py
+
+Suite Setup    suite_setup
+
+Test Teardown    cleanup_resources
+
+*** Test Cases ***
+Replica Rebuilding With Replica Deletion
```

Comment:
The test name should be `Replica Deletion While Replica Rebuilding`, which can express  the `timing`.

Reply:
The test name is following https://confluence.suse.com/pages/viewpage.action?spaceKey=LON&title=Negative+Testing. So probably the document needs to be refined? @khushboo-rancher